deps/wazuh_testing/wazuh_testing/scripts/simulate_agents.py

```diff
@@ -0,0 +1,157 @@
+
+    try:
+        start(injectors)
+        sleep(run_duration)
+    finally:
+        stop(injectors)
+
+
+def start(agent_injectors):
+    logging.info(f"Running {len(agent_injectors)} injectors...")
+    for injector in agent_injectors:
+        injector.run()
+
+
+def stop(agent_injectors):
+    logging.info(f"Stopping {len(agent_injectors)} injectors...")
+    for injector in agent_injectors:
+        injector.stop_receive()
+
+
+def main():
+    arg_parser = argparse.ArgumentParser()
+
+    arg_parser.add_argument('-a', '--manager', metavar='<manager_ip_address>', type=str, required=False,
+                            default='localhost', help='Manager IP address', dest='manager_addr')
+
+    arg_parser.add_argument('-e', '--eps', metavar='<events_per_second>', type=int, required=False, default=1000,
+                            help='Number of events per second to be generated by each agent', dest='eps')
+
+    arg_parser.add_argument('-n', '--agents', metavar='<agents_number>', type=int, default=5, required=False,
+                            help='Number of agents to create and run', dest='n_agents')
+
+    arg_parser.add_argument('-b', '--batch', metavar='<agents_batch>', dest='agent_batch',
+                            type=int, required=False, default=2, help='Number of agents to create on each process')
+
+    arg_parser.add_argument('-o', '--os', metavar='<os>', dest='os',
+                            type=str, required=False, default='debian8', help='Agent operating system',)
+
+    arg_parser.add_argument('-p', '--protocol', metavar='<protocol>', dest='agent_protocol',
+                            type=str, required=False, default=TCP, help='Communication protocol')
+
+    arg_parser.add_argument('-t', '--time', metavar='<monitoring_time>', dest='duration',
+                            type=int, required=False, default=20, help='Time in seconds for monitoring')
+
+    arg_parser.add_argument('-v', '--version', metavar='<version>', dest='version',
+                            type=str, required=False, default='4.2.0', help='Agent wazuh version', )
+
+    arg_parser.add_argument('-m', '--modules', dest='modules', required=False, type=str, nargs='+', action='store',
+                            default=['fim'], help='Active module separated by whitespace.')
+
+    arg_parser.add_argument('-s', '--modules-eps', dest='modules_eps', required=False, type=int, nargs='+',
+                            action='store', default=None, help='Active module EPS separated by whitespace.')
+
+    args = arg_parser.parse_args()
+
+    if args.agent_batch > 2:
+        logging.warning("Launching more than 2 agents per process is not advisable as Python's GIL dramatically "
+                        "reduces the performance of the agent_simulator tool when there are multiple agents running in "
+                        "the same process.")
+
+    # Calculate modules EPS
+    if args.modules is not None and args.modules_eps is not None:
```

Comment:
`args.modules` is never going to be None since it is initialized with `['fim']`.

Either you set a default value for `args.modules_eps` or you remove the default value for `args.modules`

Reply:
Sure https://github.com/wazuh/wazuh-qa/pull/1148/commits/57e715ec0b50b5962b693361f56d3df383006a55